README.md

```diff
@@ -115,3 +115,12 @@ I would like to thank Microverse Community for inspiring this project and docume
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [OPTIONAL] Well done for fixing the LICENSE 🥇 I noticed that some sections are missing from your README such as **_Tech Stack, Key Features & Future Features_**. Also, a section description for Show your support section is present but the section heading is missing 😓 Could you please fix this for documenting a professional README. As an example, you can go through [this README](https://github.com/KanzaTahreem/OOP-School-Library) 🧾